src/Zope2/Startup/tests/test_starter.py

```diff
@@ -108,6 +108,23 @@ def testSetupConflictRetries(self):
         root_wsgi_handler(conf)
         self.assertEqual(HTTPRequest.retry_max_count, 25)
 
+    def test_webdav_source_port(self):
+        from ZPublisher import WSGIPublisher
+
+        # If no value is provided, the default is 0
+        conf = self.load_config_text("""
+            instancehome <<INSTANCE_HOME>>""")
+        starter = self.get_starter(conf)
+        starter.setupPublisher()
+        self.assertEqual(WSGIPublisher._WEBDAV_SOURCE_PORT, 0)
+
+        conf = self.load_config_text("""
+            instancehome <<INSTANCE_HOME>>
+            webdav-source-port 9800""")
+        starter = self.get_starter(conf)
+        starter.setupPublisher()
```

Comment:
Same here.

Reply:
A added a cleanup step at the end of the test